src/App.js

```diff
@@ -234,6 +246,68 @@ export const App = () => {
 					"verb": operation,
 				}
 			}]
+		} else if (type === 'object' && params.objectId) {
+			body = [{
+				"object": [{
+					"operation": 'GET',
+					"action": "ALLOW",
+					"filters": [{
+						"headerType": "OBJECT",
+						"key": "$Object:objectID",
+						"matchType": "STRING_EQUAL",
+						"value": params.objectId,
+					}],
+					"targets": [{
+						"role": "OTHERS",
+						"keys": []
+					}]
+				}, {
+					"operation": 'HEAD',
+					"action": "ALLOW",
+					"filters": [{
+						"headerType": "OBJECT",
+						"key": "$Object:objectID",
+						"matchType": "STRING_EQUAL",
+						"value": params.objectId,
+					}],
+					"targets": [{
+						"role": "OTHERS",
+						"keys": []
+					}]
+				}, {
+					"operation": 'GET',
+					"action": "DENY",
+					"filters": [],
+					"targets": [{
+						"role": "OTHERS",
+						"keys": []
+					}]
+				}, {
+					"operation": 'HEAD',
+					"action": "DENY",
+					"filters": [],
+					"targets": [{
+						"role": "OTHERS",
+						"keys": []
+					}]
+				}, {
+					"operation": 'PUT',
+					"action": "DENY",
+					"filters": [],
+					"targets": [{
+						"role": "OTHERS",
+						"keys": []
+					}]
+				}, {
+					"operation": 'DELETE',
+					"action": "DENY",
+					"filters": [],
+					"targets": [{
+						"role": "OTHERS",
+						"keys": []
+					}]
+				}]
```

Comment:
Let's also add three rules forbidding `SEARCH, RANGE, RANGEHASH`. Token can be misused otherwise.